lib/static/new-ui/features/visual-checks/components/VisualChecksPage/index.module.css

```diff
@@ -0,0 +1,8 @@
+.card {
+    height: calc(100vh - 20px);
+}
+
+.card__title {
+    margin-bottom: 8px !important;
```

Comment:
why are you using `!important` here? isn't it enough just to increase the weight of the selector by nesting them (`.card .cart__title`)?